.github/workflows/ci-nuxtjs.yml

```diff
@@ -89,38 +89,38 @@ jobs:
           bucket-name-main: 'pie-aperture'
       secrets: inherit
 
-  browser-tests:
-    needs: 'build-deploy-nuxt-app-ubuntu-node20'
-    name: Run WebDriverIO Tests
-    runs-on: ubuntu-latest
-    steps:
-      # Checkout the Repo
-      - name: Checkout
-        uses: actions/checkout@v3
+  # browser-tests:
```

Comment:
Is it worth adding a comment to explain why this is commented out?

Reply:
I think it's fine to leave, Ben is actively working on this atm to reenable